zingolib/src/wallet/data.rs

```diff
@@ -1145,9 +1165,45 @@ impl TransactionMetadata {
         self.value_spent_by_pool().iter().sum()
     }
 
-    pub fn value_outgoing(&self) -> u64 {
+    pub fn value_outgoing<P: Parameters>(&self, wc: &WalletCapability, params: &P) -> u64 {
+        let viewkey = wc.ufvk().unwrap();
         self.outgoing_tx_data
             .iter()
+            .filter(
```

Comment:
after some study, i have concluded this filters away send to self and only passes outgoing transactions. if thats true, can we add a comment explaining it?

Reply:
Done!